app/models/grda_warehouse/cohorts/cohort_column.rb

```diff
@@ -0,0 +1,167 @@
+# frozen_string_literal: true
+
+module GrdaWarehouse::Cohorts
+  class CohortColumn < GrdaWarehouseBase
+    validates :class_name, presence: true, uniqueness: true
+
+    scope :active, -> { where(active: true) }
+
+    def activate
+      update(active: true)
+    end
+
+    def deactivate
+      update(active: false)
+      remove_from_cohorts
+    end
+
+    def remove_from_cohorts
+      GrdaWarehouse::Cohort.all.each do |cohort|
+        cohort.update!(column_state: cohort.column_state.reject { |col| col.cohort_column.class_name == class_name })
+      end
+    end
+
+    def self.maintain!
+      existing_class_names = all.pluck(:class_name)
```

Comment:
```suggestion
      existing_class_names = all.pluck(:class_name).to_set
```

will be significantly more performant